piecrust-uplink/README.md

```diff
@@ -0,0 +1,74 @@
+# π-crust Uplink
+
+[![Repository](https://img.shields.io/badge/github-piecrust-blueviolet?logo=github)](https://github.com/dusk-network/piecrust)
+![Build Status](https://github.com/dusk-network/piecrust/workflows/build/badge.svg)
+[![Documentation](https://img.shields.io/badge/docs-piecrust-blue?logo=rust)](https://docs.rs/piecrust-uplink/)
+
+Piecrust Uplink is the library that allows you to build smart contracts directly on top of Dusk's Piecrust virtual machine. 
+
+## Usage
+
+`Piecrust::Uplink` allows users of the contract platform to manage the interface and state with the host environment of the modules. The example below describes a barebones module. For more detailed examples, see the [modules](https://github.com/dusk-network/piecrust/tree/main/modules) folder.
```

Comment:
```suggestion
The library allows users of the contract platform to manage the interface and state with the host environment of the modules. The example below describes a barebones module. For more detailed examples, see the [modules](https://github.com/dusk-network/piecrust/tree/main/modules) folder.
```

Reply:
> I'm not sure this is the way to go, but at least the stylization is a little weird.

Agreed. Used your suggestion. It should suffice for now.